Signal/src/AppDelegate.m

```diff
@@ -160,6 +161,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     DDLogInfo(@"%@ application: didFinishLaunchingWithOptions completed.", self.tag);
 
     [OWSAnalytics appLaunchDidBegin];
+    [OWSProfilesManager sharedManager];
```

Comment:
I've added a no-op `appLaunchDidBegin` method with a comment to make the intent more clear.